tracker.js

```diff
@@ -35,6 +36,33 @@ function track() {
         if (vcapApplication.application_uris) {
             event.application_uris = vcapApplication.application_uris;
         }
+        if (process.env.VCAP_SERVICES) {
+            // refer to http://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES
+            vcapServices =  JSON.parse(process.env.VCAP_SERVICES);
+            if(Object.keys(vcapServices).length > 0) {
+                event.bound_vcap_services = {};
```

Comment:
Never mind my above comment. I didn't realize that this new `if` statement was nested within the other `if` statement.